dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -213,43 +213,31 @@ StaticLinkageCheckReport findLinkageErrors() {
     return StaticLinkageCheckReport.create(jarLinkageReports);
   }
 
-  private JarLinkageReport findInvalidReferences(Path jarPath, SymbolsInFile symbolsInFile) {
+  private JarLinkageReport findInvalidReferences(Path jarPath, SymbolReferenceSet symbolReferenceSet) {
     JarLinkageReport.Builder reportBuilder = JarLinkageReport.builder().setJarPath(jarPath);
 
     // TODO(suztomo): implement validation for field, method and class references in the table
     return reportBuilder.build();
   }
 
   /**
-   * Scans class files in the jar file and returns a {@link SymbolsInFile} populated with symbolic
-   * references.
+   * Scans class files in the jar file and returns a {@link SymbolReferenceSet} populated with
+   * symbolic references.
    *
    * @param jarFilePath absolute path to a jar file
    * @return symbol references and classes defined in the jar file
    */
-  static SymbolsInFile scanSymbolReferences(Path jarFilePath) {
-    Preconditions.checkState(
+  static SymbolReferenceSet scanSymbolReferencesInJar(Path jarFilePath) {
+    Preconditions.checkArgument(
```

Comment:
FYI I find it more readable to statically import any `Preconditions` static method I use. (Same with `Truth.assertThat()`.)

Reply:
Updated to use static import.
`assertThat` would conflict hamcrest's Assert. So I'd keep as they're.

Reply:
That's okay. But I think you can statically import a method from more than one class, and Java will find the overload that matches the argument.

Reply:
Good to know that. I confirmed it works:

```
import static org.junit.Assert.assertThat;
import static org.hamcrest.CoreMatchers.is;
import static com.google.common.truth.Truth.assertThat;

import com.google.common.collect.ImmutableList;
import org.junit.Test;

public class FooTest {
  @Test
  public void testStaticImports() {
    assertThat("This is test", "foo", is("foo"));
    assertThat(ImmutableList.of()).isEmpty();
  }
}
```